docs/the_black_code_style/current_style.md

```diff
@@ -166,7 +166,7 @@ If you use Flake8, you have a few options:
    max-line-length = 80
    ...
    select = C,E,F,W,B,B950
-   extend-ignore = E203, E501, E704
+   extend-ignore = E203, E501, E701, E704
```

Comment:
I feel like this whole section should be replaced with a link to the "using Black with other tools" page.

Reply:
I can do these, working on them now.